pkg/proto/datadog/trace/idx/span.proto

```diff
@@ -0,0 +1,153 @@
+syntax = "proto3";
+
+package datadog.trace.idx;
+
+option go_package="pkg/proto/pbgo/trace/idx";
+
+message AnyValue {
+	oneof value {
+		// @gotags: json:"string_value" msg:"string_value"
+		uint32 stringValue = 1;
```

Comment:
Maybe we can be more descriptive about this field name, making it more obvious that it's a String reference/key/othername? (eg `stringRef` or similar)

It would also leave room, if needed in the future, to have an actual `string stringValue` (just in case)